stefaniaPopescu26/assignment3/DictionaryTest.java

```diff
@@ -0,0 +1,43 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```

Comment:
Avoid importing a whole package. It is better to just import exactly what you are using otherwise you might end up with double imports that you won't even be aware of.